SmartDeviceLink/SDLPresentChoiceSetOperation.m

```diff
@@ -129,7 +129,7 @@ - (void)sdl_updateKeyboardPropertiesWithCompletionHandler:(nullable void(^)(void
 - (void)sdl_presentChoiceSet {
     __weak typeof(self) weakself = self;
     [self.connectionManager sendConnectionRequest:self.performInteraction withResponseHandler:^(__kindof SDLRPCRequest * _Nullable request, __kindof SDLRPCResponse * _Nullable response, NSError * _Nullable error) {
-        if (error != nil) {
+        if (!response.success.boolValue || error != nil) {
```

Comment:
Checking `response.success.boolValue` does not work because `success` is `true` in the HMI bug. Rather  the `response.resultCode` should be checked to make sure that it is `SUCCESS`.

Reply:
Well, the HMI bug is fixed, so perhaps we should just close this issue unless it crops up elsewhere?

Reply:
This is the first time I've seen this HMI bug so I agree that we should close this issue and PR.